pycalphad/io/database.py

```diff
@@ -465,3 +472,13 @@ def search(self, query):
         >>>> db.search(where('eid') == eid)
         """
         return self._parameters.search(query)
+
+    def _parameter_queue_process(self):
```

Comment:
I think `_process_parameter_queue()` could be a better name, since it's clearer about what calling the method actually does. Thoughts?